cmd/interceptors/main.go

```diff
@@ -92,14 +92,24 @@ func main() {
 		return
 	}
 
-	clusterInterceptorList, err := clusterinterceptorsinformer.Get(ctx).Lister().List(labels.NewSelector())
-	if err != nil {
-		return
-	}
-
-	if err := service.UpdateCRDWithCaCert(ctx, tc.TriggersV1alpha1(), clusterInterceptorList, caCert); err != nil {
+	if err := listAndUpdateClusterInterceptorCRD(ctx, tc, service, caCert); err != nil {
 		return
 	}
+	ticker := time.NewTicker(time.Minute)
```

Comment:
time.Minute might be a bit too high for the initial cert fetch. maybe 10 * time.Second?

Reply:
Initial cert fetch are done [here](https://github.com/tektoncd/triggers/pull/1397/files#diff-ffaa87bda08175ee16b659d2b32cc08cb3a71e1f24797ba38b5d32619f76b963R95) before starting ticker so kept 1 minute to sync/update if there are any core interceptor found without `caBundle` .

Reply:
Ah I missed that. Thank you.